dts/arm/nordic/nrf51822.dtsi

```diff
@@ -5,6 +5,15 @@
 #include <dt-bindings/gpio/gpio.h>
 #include "nrf5_common.dtsi"
 
+#include <nordic/override.h>
+/*
+ * This value can be overridden at the board level or in an application specific
+ * dts/nordic/override.h file.
+ */
+#ifndef DT_DEFAULT_IRQ_PRIORITY
```

Comment:
Not hardcore about it, but maybe the `DT_` prefix could be left out, so that `DT_` is reserved for macros automatically generated by the devicetree scripts, similar to how `CONFIG_` is reserved for Kconfig.

Reply:
`DEFAULT_DT_IRQ_PRIORITY` might work too.

Reply:
I initially thought about not prefixing this macro with `DT_` but then I saw https://github.com/zephyrproject-rtos/zephyr/blob/d4ba8fff66bd39f413a853956aafd4609041cec3/dts/common/mem.h#L10
and got a bit confused. Is it stated somewhere that `DT_` prefix is only for the generated stuff (and related fixups)?

I like the `DEFAULT_DT_IRQ_PRIORITY` proposal. Can use it if there are no objections.

Reply:
> Is it stated somewhere that `DT_` prefix is only for the generated stuff (and related fixups)?

Nothing written down. Had forgotten that fixup files also #define `DT_` stuff. Maybe that could be cleaned up a bit at some point too.

Guess there might be less risk of confusion for function-like macros like `DT_SIZE_K`, though I wouldn't mind changing that as well.